cg/services/order_validation_service/workflows/mip_dna/validation_rules.py

```diff
@@ -1,3 +1,47 @@
+from cg.services.order_validation_service.rules.case_sample.rules import (
+    validate_application_compatibility,
+    validate_application_exists,
+    validate_application_not_archived,
+    validate_buffer_skip_rc_condition,
+    validate_concentration_interval_if_skip_rc,
+    validate_concentration_required_if_skip_rc,
+    validate_fathers_are_male,
+    validate_fathers_in_same_case_as_children,
+    validate_mothers_are_female,
+    validate_mothers_in_same_case_as_children,
+    validate_pedigree,
+    validate_sample_names_not_repeated,
+    validate_samples_exist,
+    validate_sex_required_for_new_samples,
+    validate_source_required,
+    validate_status_required_if_new,
+    validate_subject_ids_different_from_case_names,
+    validate_subject_ids_different_from_sample_names,
+    validate_volume_interval,
+    validate_wells_contain_at_most_one_sample,
+)
+
 CASE_RULES: list[callable] = []
```

Comment:
Where are the case rules?

Reply:
https://github.com/Clinical-Genomics/cg/pull/3625/commits/22b77ca773c6e0c4f5ba5235ed4c3ddf20551148